extension/chrome/elements/reply_pubkey_mismatch.ts

```diff
@@ -63,7 +63,7 @@ Catch.try(async () => {
     renderHelpDialog: () => undefined,
     renderSendingAddrDialog: () => undefined,
     closeMsg: () => undefined,
-    factoryAtt: (att) => `<div>${att.name}</div>`,
+    factoryAtt: (att) => `<div>${att.name}</div>`, // xss-escaped
```

Comment:
I meant actually escape it: `<div>${Xss.escape(att.name)}</div>`

Reply:
oh, sorry, corrected